api/src/utils/index.ts

```diff
@@ -195,24 +195,25 @@ export const parseDistanceFromGoogleApi = (data: GoogleDistance) => {
  * @param entity
  * @param dto
  */
-export const isSameBookingDate = (entity: BookingDateEntity, dto: BookingDateDto) =>   
-  isSameDay(new Date(entity.date), new Date(dto.date)) && entity.period === dto.period && entity.arrivalTime === dto.arrivalTime;
+export const isSameBookingDate = (entity: BookingDateEntity, dto: BookingDateDto) =>
+  isSameDay(new Date(entity.date), new Date(dto.date)) &&
+  entity.period === dto.period &&
+  entity.arrivalTime === dto.arrivalTime;
 
-  /**
+/**
  * sort array of booking dates by date
  * @param BookingDate[]
  * @param BookingDate[]
  */
 export const sortBookingDates = (a: BookingDateEntity, b: BookingDateEntity) => {
   console.dir(getUnixTime(a.date));
   console.dir(getUnixTime(b.date));
```

Comment:
logalogs

Reply:
ha, you want me to remove the logs?
saw you are the author.
![image](https://user-images.githubusercontent.com/46575401/119181028-ebf21800-ba25-11eb-87d3-52c7ee357ea6.png)